src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/CassandraEventSchema.scala

```diff
@@ -0,0 +1,133 @@
+package com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra
+
+import java.util.Date
+
+import com.microsoft.partnercatalyst.fortis.spark.dto.FortisEvent
+import com.microsoft.partnercatalyst.fortis.spark.sinks.cassandra.dto._
+import java.text.Collator
+import java.util.Locale
+
+import com.microsoft.partnercatalyst.fortis.spark.analyzer.timeseries.{Period, PeriodType}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.gender.GenderDetector.{Female, Male}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.sentiment.SentimentDetector.Neutral
+
+object CassandraEventSchema {
+  def apply(item: FortisEvent, batchid: String): Event = {
+    Event(
+      pipelinekey = item.details.pipelinekey,
+      externalsourceid = item.details.externalsourceid,
+      computedfeatures = Utils.getFeature(item),
+      eventtime = item.details.eventtime,
+      batchid = batchid,
+      eventlangcode = item.analysis.language.orNull,
+      eventid = item.details.eventid,
+      insertiontime = new Date().getTime,
+      body = item.details.body,
+      sourceurl = item.details.sourceurl,
+      title = item.details.title)
+  }
+}
+
+object CassandraPopularPlaces {
+  def apply(item: EventBatchEntry): Seq[PopularPlaceAggregate] = {
+    for {
+      kw <- Utils.getConjunctiveTopics(Option(item.computedfeatures.keywords))
+      location <- item.computedfeatures.places
+      periodType <- Utils.getCassandraPeriodTypes
+    } yield PopularPlaceAggregate(
+      pipelinekey = item.pipelinekey,
+      centroidlat = location.centroidlat,
+      centroidlon = location.centroidlon,
+      placeid = location.placeid,
+      periodstartdate = Period(item.eventtime.getTime, periodType).startTime(),
+      periodenddate = Period(item.eventtime.getTime, periodType).endTime(),
+      periodtype = periodType.periodTypeName,
+      period = periodType.format(item.eventtime.getTime),
+      externalsourceid = item.externalsourceid,
+      mentioncount = item.computedfeatures.mentions,
+      conjunctiontopic1 = kw._1,
+      conjunctiontopic2 = kw._2,
+      conjunctiontopic3 = kw._3,
+      avgsentiment = item.computedfeatures.sentiment.neg_avg
+    )
+  }
+}
+
+object CassandraEventTagsSchema {
+  def apply(item: EventBatchEntry): Seq[EventTags] = {
+    for {
+      kw <- item.computedfeatures.keywords
+      location <- item.computedfeatures.places
+    } yield EventTags(
+      pipelinekey = item.pipelinekey,
+      centroidlat = location.centroidlat,
+      centroidlon = location.centroidlon,
+      eventid = item.eventid,
+      topic = kw.toLowerCase,
+      eventtime = item.eventtime.getTime,
+      externalsourceid = item.externalsourceid,
+      placeid = location.placeid
+    )
+  }
+}
+
+object Utils {
```

Comment:
I considered that but I have a couple of unit tests to validate the functionality. How have we approached a similar situation in the past with the project?